packages/angular/cli/lib/config/schema.json

```diff
@@ -27,7 +27,7 @@
     "projects": {
       "type": "object",
       "patternProperties": {
-        "^[a-zA-Z][.0-9a-zA-Z]*(-[.0-9a-zA-Z]*)*$": {
+        "^(?:@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*$": {
```

Comment:
The `/` should probably be escaped

```
^(?:@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$
```

Also can you add the below footer to your commit

```

Closes #14797
```

Thanks and sorry for the delay.

Reply:
Never mind. How long will Lvy be released? I'm looking forward to it.